source/apps/environment.txt

```diff
@@ -102,6 +102,8 @@ Before You Begin
 
       - .. include:: /includes/prereqs/app-services-cli.rst
 
+      - .. include:: /includes/prereqs/app-id.rst
```

Comment:
not for this PR, but the TOC on this page is funky. gotta set up a lvl

<img width="546" alt="Screen Shot 2023-01-25 at 4 32 43 PM" src="https://user-images.githubusercontent.com/90647379/214695893-a990e4cf-f8c9-42cd-bf48-551c362b3d5e.png">